libs/auth-api-lib/src/lib/clients/admin/dto/admin-create-client.dto.ts

```diff
@@ -11,12 +12,13 @@ export class AdminCreateClientDto {
   readonly clientId!: string
 
   @IsNotEmpty()
-  @IsEnum(AdminClientType)
+  @IsEnum(ClientType)
   @ApiProperty({
     example: 'spa',
-    enum: AdminClientType,
+    enum: ClientType,
+    enumName: 'ClientType',
   })
-  readonly clientType!: AdminClientType
+  readonly clientType!: ClientType
```

Comment:
Should make sure we have and test server-side validation for `clientType` and `clientId`.